src/config_ctrl.ts

```diff
@@ -30,13 +40,38 @@ export class LogzioConfigCtrl {
     this.current.database = def.example || 'es-index-name';
   }
 
-  addHeader() {
-    var addIndex = this.current.jsonData.headers.length;
-
-    this.current.jsonData.headers.splice(addIndex, 0, { key : "", value: "" });
+  getSuggestUrls() {
+    return [
+      'https://api.logz.io/v1/elasticsearch',
+      'https://api-eu.logz.io/v1/elasticsearch',
+    ];
   }
 
-  removeHeader($index) {
-    this.current.jsonData.headers.splice($index, 1);
+  migrateUrlAndApiKey(instanceSettings) {
+    if (
+      (!instanceSettings.jsonData.url ||
```

Comment:
```suggestion
                    if (!instanceSettings.jsonData.url)
```
I think the other parts are irrelevant, why is there a need to check the length, instead of just checking if the string is empty by a boolean operation (like the one suggested?)